perception/autoware_detected_object_validation/config/object_lanelet_filter.param.yaml

```diff
@@ -19,3 +19,5 @@
         enabled: false
         velocity_yaw_threshold: 0.785398 # [rad] (45 deg)
         object_speed_threshold: 3.0 # [m/s]
+      debug: false
+      lanelet_extra_margin: 0.0
```

Comment:
@sebekx 

Could you update README and add the explanation for these params?

Reply:
updated readme